docs/configuration.md

```diff
@@ -1505,6 +1505,16 @@ Apart from these, the following properties are also available, and may be useful
     <code>spark.rpc.lookupTimeout</code> if they are not configured.
   </td>
 </tr>
+<tr>
+  <td><code>spark.network.io.preferDirectBufs</code></td>
+  <td>true</td>
+  <td>
+    If enable then off-heap buffer allocations are prefered by the shared ByteBuf allocators.
```

Comment:
On a last look, there are probably a few small typos to change in the text, but they're not a big deal: enable -> enabled, prefered -> preferred, ByteBuf -> ByteBuffer. Should this be prefixed with "(Netty only)" like similar properties, in the description? I don't know if we really need to mention Netty as it's an implementation detail, and would be just as happy to remove "Netty" from all the descriptions here.

Reply:
Instead of replacing ByteBuf with ByteBuffer I have just removed it (as ByteBuf and ByteBuffer is really two separate things: ByteBuf is Netty related and ByteBuffer is nio related).